app/leaderboard/page.tsx

```diff
@@ -285,6 +302,62 @@ export default function LeaderboardPage() {
           </>
         )}
 
+        {activeTab === "talent" && (
+          <>
+            {loading && (
+              <div className="space-y-3">
+                {Array.from({ length: 5 }).map((_, i) => (
+                  <div key={i} className="flex items-center gap-3 p-3">
+                    <Skeleton className="h-8 w-8 rounded-full" />
+                    <div className="flex-1 space-y-1">
+                      <Skeleton className="h-4 w-32" />
+                      <Skeleton className="h-3 w-24" />
+                    </div>
+                    <div className="space-y-1 text-right">
+                      <Skeleton className="h-4 w-12" />
+                      <Skeleton className="h-3 w-16" />
+                    </div>
+                  </div>
+                ))}
+              </div>
+            )}
+
+            {!loading && (
+              <>
+                {/* Filter for boosted creators only */}
+                {top200Entries.filter(
+                  (entry) => entry.isBoosted && entry.score > 0,
+                ).length === 0 ? (
+                  <div className="text-center py-8">
+                    <p className="text-muted-foreground">
+                      No boosted creators found
+                    </p>
+                  </div>
+                ) : (
+                  /* Talent leaderboard list */
+                  <CreatorList
+                    items={top200Entries
+                      .filter((entry) => entry.isBoosted && entry.score > 0)
+                      .map((user) => ({
+                        id: user.id,
+                        name: user.name,
+                        avatarUrl: user.pfp,
+                        rank: user.rank,
+                        primaryMetric: getUsdcRewards(user.score, user.rank),
+                        secondaryMetric: `Creator Score: ${user.score.toLocaleString()} | Tokens: ${(user.tokenBalance || 0).toLocaleString()}`,
+                      }))}
+                    onItemClick={(item) => {
+                      // Navigate to profile page
+                      window.location.href = `/${item.id}`;
```

Comment:
It's probably better to use next router for this so we benefit from their page transitions by default.